### PR TITLE
Project renovations

### PR DESCRIPTION
**Pull request by armcknight**
Just some housekeeping, including adding some file/folder references to Example.xcworkspace, to facilitate authoring of e.g. readmes and config files in Xcode, alongside the Swift sources. That includes a folder reference to `Pod/`, to get all the subdirectory readmes, etc.

Screen Shot 2019-04-23 at 2 52 09 PM

Wasn't sure who all to add to authors, but I started by including myself and folks I've already chatted with for this repo. Other suggestions welcome.

### CODE REVIEW

**Comment by minimusic** [comment]
Would rather not see the top level readme being duplicated in the file tree (also in Podspec Metadata)

**Comment by minimusic** [comment]
The pod/pods naming might be unnecessarily confusing. Your suggestion of "Sources" is an improvement, but still overlaps all the files in the "Development Pods" directory. I don't have a specific suggestions to allow readme editing without any duplication, though; I'd like more opinions from the rest of the team.

**Comment by minimusic** [comment]
I'm not thrilled about it, but maybe the most straight forward solution (suggested by Chris B) is to combine all of our ReadMes into just the single, top-level one... But I'd also be okay deferring that to another PR, and strip all the file-system stuff out of this one so we can merge your other fixes/updates.

**Comment by armcknight** [comment]
Hey y'all! Sorry for neglecting this... 

I removed the folder reference to the pod source directory and top level readme, but left the other config files like podspec and gemfile. 

I agree about the duplication of the file references in Development Pods with the folder reference... I have an open issue and wip pr to include all subdirectory READMEs in the Pods project as well: https://github.com/armcknight/CocoaPods/pull/1

Let me know if we'd like to move ahead with renaming `Pod` to `Sources`.

File: Swiftilities.podspec [hunk 1]
```diff
@@ -12,7 +12,12 @@ Pod::Spec.new do |s|
 
   s.homepage         = "https://github.com/rightpoint/Swiftilities"
   s.license          = 'MIT'
-  s.author           = { "Nicholas Bonatsakis" => "nick.bonatsakis@raizlabs.com" }
+  s.authors          = {
+                         "Chris Ballinger" => "chris.ballinger@raizlabs.com",
+                         "Nicholas Bonatsakis" => "nick.bonatsakis@raizlabs.com",
+                         "Zev Eisenberg" => "zev.eisenberg@raizlabs.com",
```
Comment (ZevEisenberg) [inline Swiftilities.podspec]: Nick’s Raizlabs address is probably dead. These should all be Rightpoint, and Nick should either be removed or have an updated address

Comment (minimusic) [reply Swiftilities.podspec]: +1 Good call.

Comment (armcknight) [reply Swiftilities.podspec]: Done!

**Comment by minimusic** [comment]
Re-tested; still looks good to me! You could add more authors from the repo contributors list, but I'm ambivalent about it.

**Pull request merged**